ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -195,29 +206,28 @@
 {%         set row_default_encapsulation = router_bgp.address_family_evpn.neighbor_default.encapsulation | arista.avd.default("vxlan") %}
 {%         set row_nhs_source_interface = router_bgp.address_family_evpn.neighbor_default.next_hop_self_source_interface | arista.avd.default('-') %}
 | {{ row_default_encapsulation }} | {{ row_nhs_source_interface }} |
-{%     endif%}
-{%     if router_bgp.address_family_evpn.evpn_hostflap_detection is arista.avd.defined %}
+{%         endif%}
+{%         if router_bgp.address_family_evpn.evpn_hostflap_detection is arista.avd.defined %}
 
 #### EVPN Host Flapping Settings
 
 | State | Window | Threshold |
 | ----- | ------ | --------- |
-{%         if router_bgp.address_family_evpn.evpn_hostflap_detection.enabled | arista.avd.default(true) == true %}
-{%             set evpn_hostflap_detection_state = "Enabled" %}
-{%             set evpn_hostflap_detection_window = router_bgp.address_family_evpn.evpn_hostflap_detection.window | arista.avd.default('-') %}
-{%             set evpn_hostflap_detection_threshold = router_bgp.address_family_evpn.evpn_hostflap_detection.threshold | arista.avd.default('-') %}
-{%         else %}
-{%             set evpn_hostflap_detection_state = "Disabled" %}
-{%             set evpn_hostflap_detection_window = '-' %}
-{%             set evpn_hostflap_detection_threshold = '-' %}
-{%         endif %}
+{%             if router_bgp.address_family_evpn.evpn_hostflap_detection.enabled | arista.avd.default(true) %}
+{%                 set evpn_hostflap_detection_state = "Enabled" %}
+{%                 set evpn_hostflap_detection_window = router_bgp.address_family_evpn.evpn_hostflap_detection.window | arista.avd.default('-') %}
+{%                 set evpn_hostflap_detection_threshold = router_bgp.address_family_evpn.evpn_hostflap_detection.threshold | arista.avd.default('-') %}
```

Comment:
The logic is good but we can move both of the variables assignment, ```evpn_hostflap_detection_window```,  ```evpn_hostflap_detection_threshold``` outside the if else condition and set with a ```default``` value instead of defining them twice in ```if``` and ```else``` blocks. This will reduce the number of lines.

We can keep them below endif.